### ISSUE TITLE
make command throws error

### ISSUE THREAD

**Issue by tobiasbueschel** [UCL/ucl-latex-thesis-templates#9]
I'm getting the following error when running `make all`

```bash
lualatex --halt-on-error Main.tex
This is LuaTeX, Version 0.95.0 (TeX Live 2016)
 restricted system commands enabled.
(./Main.tex
LaTeX2e <2016/03/31>
Babel <3.9r> and hyphenation patterns for 1 language(s) loaded.
(/usr/local/texlive/2016/texmf-dist/tex/latex/nag/nag.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/nag/nag-l2tabu.cfg)
(/usr/local/texlive/2016/texmf-dist/tex/latex/nag/nag-orthodox.cfg))
(./ucl_thesis.cls
Document Class: ucl_thesis 2014/08/12 v1.6 UCL MRes/MPhil/PhD Thesis class
(/usr/local/texlive/2016/texmf-dist/tex/latex/base/book.cls
Document Class: book 2014/09/29 v1.4h Standard LaTeX document class
(/usr/local/texlive/2016/texmf-dist/tex/latex/base/bk12.clo))
(/usr/local/texlive/2016/texmf-dist/tex/latex/psnfss/times.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/psnfss/mathptmx.sty))
(./MainPackages.tex
(/usr/local/texlive/2016/texmf-dist/tex/latex/blindtext/blindtext.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/tools/xspace.sty))
(/usr/local/texlive/2016/texmf-dist/tex/latex/emptypage/emptypage.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/graphicx.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/keyval.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/graphics.sty
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics/trig.sty)
(/usr/local/texlive/2016/texmf-dist/tex/latex/graphics-cfg/graphics.cfg)
(/usr/local/texlive/2016/texmf-dist/tex/lualatex/luatex-def/luatex.def
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/infwarerr.sty)
(/usr/local/texlive/2016/texmf-dist/tex/generic/oberdiek/ltxcmds.sty))))
! Undefined control sequence.
l.21 \pdfimageresolution
                      =300
 317 words of node memory still in use:
   1 hlist, 1 rule, 1 dir, 4 glue, 40 glue_spec, 1 write nodes
   avail lists: 2:12,3:1,4:1,5:1,7:2,9:3
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on Main.log.
make: *** [Main.pdf] Error 1
```

Would you know what might be the cause of it? Maybe related to:
```bash
! Undefined control sequence.
l.21 \pdfimageresolution
                      =300
```

**Comment by ikirker** [comment]
Some update to LuaTeX broke it, as far as I can tell. :(

Delete or comment out that line from `MainPackages.tex` and it should work.

That line is responsible for setting default resolutions for images, so some images may appear at weird sizes and have to be resized.

**Comment by ikirker** [comment]
Removed the line in commit [fd3559a6969ce76e1f5802784328a8cea341bbfd](https://github.com/UCL/ucl-latex-thesis-templates/commit/fd3559a6969ce76e1f5802784328a8cea341bbfd).

**Comment by ikirker** [comment]
Included in release v1.0.4.
